run_test.py

```diff
@@ -0,0 +1,230 @@
+import subprocess
+import argparse
+import yaml
+import sys
+import re
+import os
+
```

Comment:
please use docstring for describing the module.

Probably the most important thing is a single-line summary, eg. "run unity and pytest tests in host environment"